### ISSUE TITLE
test pg_eucjp FAILED

### ISSUE THREAD

**Issue by joelonsql** [MasaoFujii/pg_cheat_funcs#2]
Hi Fujii,

Thanks for making pg_cheat_funcs!

make installcheck fails on Ubuntu 20.04.2 LTS running Ubuntu 13.2-1.pgdg20.04+1.

Maybe I need to run some additional apt-get command to install the EUC_JP encoding before running the tests?

    $ cat regression.diffs
    diff -U3 /home/joel/postgresql-extensions/pg_cheat_funcs/expected/pg_eucjp.out /home/joel/postgresql-extensions/pg_cheat_funcs/results/pg_eucjp.out
    --- /home/joel/postgresql-extensions/pg_cheat_funcs/expected/pg_eucjp.out	2021-04-26 06:55:53.064946839 +0000
    +++ /home/joel/postgresql-extensions/pg_cheat_funcs/results/pg_eucjp.out	2021-04-26 07:00:01.184044549 +0000
    @@ -1,17974 +1,5 @@
    CREATE DATABASE regtest_cheat_funcs_eucjp ENCODING 'EUC_JP' TEMPLATE template0;
    +ERROR:  encoding "EUC_JP" does not match locale "en_US.UTF-8"
    +DETAIL:  The chosen LC_CTYPE setting requires encoding "UTF8".

**Comment by MasaoFujii** [comment]
Thanks for the report!

> Maybe I need to run some additional apt-get command to install the EUC_JP encoding before running the tests?

Probably you need to create the database cluster with locale 'C'.

This makes me think that "LOCALE 'C'" option should be specified in
"CREATE DATABASE" command in the regression test. If I can confirm
this change is valid, I will apply it.

**Comment by joelonsql** [comment]
ども ありがと ございます for the prompt reply.

I can confirmed your proposed fix works just as you predicted.
The regression.diffs now only differs for the CREATE DATABASE command I changed, which is expected.

    $ cat regression.diffs
    diff -U3 /home/joel/postgresql-extensions/pg_cheat_funcs/expected/pg_eucjp.out /home/joel/postgresql-extensions/pg_cheat_funcs/results/pg_eucjp.out
    --- /home/joel/postgresql-extensions/pg_cheat_funcs/expected/pg_eucjp.out	2021-04-26 06:55:53.064946839 +0000
    +++ /home/joel/postgresql-extensions/pg_cheat_funcs/results/pg_eucjp.out	2021-04-26 17:36:16.885622696 +0000
    @@ -1,4 +1,4 @@
    -CREATE DATABASE regtest_cheat_funcs_eucjp ENCODING 'EUC_JP' TEMPLATE template0;
    +CREATE DATABASE regtest_cheat_funcs_eucjp ENCODING 'EUC_JP' LOCALE 'C' TEMPLATE template0;
    \c regtest_cheat_funcs_eucjp
    SET client_encoding TO 'EUC_JP';
    CREATE EXTENSION pg_cheat_funcs;

**Comment by MasaoFujii** [comment]
> I can confirmed your proposed fix works just as you predicted.

Thanks a lot!

I firstly added "LOCALE 'C'" into CREATE DATABASE command in the regression test, but that option is available only in PostgreSQL 13 or later. So, to make the regression test work with also PostgreSQL 12 or before, I finally used LC_COLLATE and LC_CTYPE instead of LOCALE to specify the locale 'C'.

I pushed this change to the master branch.
